Update readme, fix typos and comments

slspencer opened the pull request:
Updated the Readme file, added a screenshot, and updated some OS logos. And fixed a typo. Testing the PR review by Copilot.

slspencer committed:
typos and comments

slspencer committed:
update Readme and logo files

slspencer committed:
updated separation lines

slspencer committed:
Updated comments

slspencer committed:
improved image path

slspencer committed:
Updated comments

DSCaskey committed:
replace missing cli options and remove redundant comments

DSCaskey reviewed (commented):
What happened with this: ?

// ... [Omitting unchanged lines for brevity—continues as previously documented]

It deleted half the commnd line options, causing the tests to fail.

DSCaskey commented:
I replaced the missing code (with comments), removed the now redundant method comments from the header file, and renamed the pointer "instance" to the more descriptive "commandLine".

DSCaskey reviewed (approved):
Tests now work.

slspencer commented:
> replaced the missing code (with comments), removed the now redundant method comments from the header file, and renamed the pointer "instance" to the more descriptive "commandLine".

Thanks @dscaskey. I was also trying copilot to add comments, and I think that the code was truncated. Good to know.
